### ISSUE TITLE
Segmentation fault on kubernetes monitor

### ISSUE THREAD

**Issue by thirunavukkarasumca** [draios/sysdig#514]
I tried to monitor a kubernetes platform which is running in google compute engine using this command "sysdig  -k https://&lt;username&gt;:&lt;password&gt;@&lt;ip&gt;:443" 

and  i got error message  "Segmentation fault (core dumped)"

**Comment by gianlucaborello** [comment]
Sounds like a crash due to some Kubernetes state that we haven't predicted. It would be very helpful if you could:

1) Make sure you're using the latest stable release, 0.6.0

2) Make sysdig crash again with gdb and get a stack trace. You can do something like:

```
# gdb --args sysdig  -D -k https://<username>:<password>@<ip>:443
$ run
... wait until crash occurs ...
$ bt
```

If the resulting stack trace will be full of `?????` and unknown symbols, please use the sysdig binary you can find at https://s3.amazonaws.com/download.draios.com/stable/tgz/x86_64/sysdig-0.6.0-x86_64.tar.gz since it is not stripped of debugging symbols.

3) Post us the output of your kubernetes state, e.g.

```
MASTER_API="https://user:password@ip:443"

curl $MASTER_API/api/v1/namespaces
curl $MASTER_API/api/v1/nodes
curl $MASTER_API/api/v1/pods
curl $MASTER_API/api/v1/replicationcontrollers
curl $MASTER_API/api/v1/services
curl $MASTER_API/api/v1/endpoints
```

Thanks

**Comment by aleks-f** [comment]
@thirunavukkarasumca : can you run sysdig debug binary and let us know if you get an assert failure

**Comment by aleks-f** [comment]
most likely due to a bug, [fixed](https://github.com/draios/sysdig/commit/d38729a334544c7a25bea59270c7a3e7b3d67ce6#diff-402658f955f2982a224cc43016be2bb0L1515) for the upcoming release

**Comment by gianlucaborello** [comment]
Should be fixed with 0.7.1
